vault/request_handling.go

```diff
@@ -1992,3 +1993,34 @@ func (c *Core) checkSSCTokenInternal(ctx context.Context, token string, isPerfSt
 	// status code.
 	return "", logical.ErrMissingRequiredState
 }
+
+func (c *Core) GetUserLockoutFromConfig(logicalType string) UserLockoutConfig {
+	conf := c.rawConfig.Load()
+	if conf == nil {
+		return UserLockoutConfig{}
+	}
+	userlockouts := conf.(*server.Config).UserLockouts
+
+	if userlockouts == nil {
+		return UserLockoutConfig{}
+	}
+
+	var commonUserLockoutConfig UserLockoutConfig
+	var authUserLockoutConfig UserLockoutConfig
```

Comment:
How about defining this within the scope of the inner `if` right before it is being used?